my-app/src/pages/UserScreenPage.tsx

```diff
@@ -0,0 +1,269 @@
+import * as React from "react";
+import "./UserScreenPage.css";
+import { FormErrors } from "./FormsErrors";
+import * as Validation from "../Validation";
+import { mutateCreatUser } from "../CreateUserIntegration";
+import { changeURL } from "../ChangeUrl";
+
+interface UserScreenState {
+  name: string;
+  phone: string;
+  birthDate: Date;
+  birthDateString: string;
```

Comment:
é a mesma coisa com formato diferente?
nao value a pena colocar no state os 2, vai precisar manter e 2 formatos atualizados sempre e um outra pessoa que for modificar o seu código pode nao saber